check_line_length.py

```diff
@@ -24,14 +26,30 @@ def check_line_length(file_path):
     interrupts execution with exit code 1 otherwise
     """
     file = file_path.split('/')[-1]
+    limit_type = 'text'
     with open(file_path) as f:
         lines = f.readlines()
     for (line_number, line) in enumerate(lines, start=1):
+        is_code = re.search(r'code-block', line)
```

Comment:
I am sure if i understand this correctly.
You are trying to check if 'code-block' is in the line?
But, the code-block might not have 'code-block' written. ( It most probably never have that. )

Code-block is:
```
Hello i am a code-block!
```

Reply:
ok tomorrow i'll create for 
```
this
```

Reply:
i see about code block from here @atb00ker
http://www.sphinx-doc.org/en/1.5/markup/code.html

Reply:
and this is not using markdown but using reStructuredText. I only see using `.. code-block::programming` for checked code example.